extensions/stats/plugin.cc

```diff
@@ -384,13 +384,10 @@ bool PluginRootContext::initializeDimensions(const json& j) {
                              .value_or(default_field_separator);
   auto value_separator = JsonGetField<std::string>(j, "value_separator")
                              .value_or(default_value_separator);
-  auto stat_prefix =
```

Comment:
Shall we mark the field deprecated, or simply remove it from the proto file? So that doc on istio.io could also be updated.

Reply:
Done.